### DISCUSSION TITLE
Mathematical constants showing

### DISCUSSION THREAD

**Discussion by a-Rahmati** [Qalculate/qalculate-gtk#252] [Q&A] [answered]
Hello,
Is there any option in Qalculate that shows mathematical constants (e, π, ...) by symbol, not a decimal number?
I mean, I prefer this✅:
![Selection_132](https://user-images.githubusercontent.com/52320192/112992470-71232400-917d-11eb-8731-2cce18d7d1e0.png)
to this☒:
![Qalculate!_131](https://user-images.githubusercontent.com/52320192/112991219-1fc66500-917c-11eb-8374-39e5aeed4841.png)

Thanks

**Comment by hanna-kn** [comment, marked as answer]
Activate exact mode (using the exact button, the menu activated when you click the parse options status ("RAD"), the result context menu, or Mode->Approximation->Exact).

**Reply by a-Rahmati** [reply]
It's working.

Thank you.
